docker/development/init.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# SET POSTGRES DETAILS
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=your-super-secret-and-long-postgres-password
+POSTGRES_DB=postgres
+POSTGRES_PORT=5432
+POSTGRES_HOST=db
+CONTAINERNAME=supabase-db
+SUPERFLOWS_PORT=8080
+
+manage_supabase_repo() {
+    if [ -d "supabase/.git" ]; then
+        echo "Supabase repository already exists. Pulling the latest changes..."
+        cd supabase && git pull && cd ..
+    else
+        echo "Cloning the Supabase repository..."
+        git clone -n --depth=1 --filter=tree:0 https://github.com/supabase/supabase && cd supabase && git sparse-checkout set --no-cone docker && git checkout && cd ..
+    fi
+}
+
+# Function to manage the .env file
+manage_env_file() {
+    env_file=".env"
+    if [ ! -f "$env_file" ]; then
+        if [ -f "../../.env.example" ] && [ -f "supabase/docker/.env.example" ]; then
+            echo "Creating .env file from .env.example files..."
+            cat "supabase/docker/.env.example" "../../.env.example" > $env_file
```

Comment:
To add docker-required env vars which aren't required when running locally

```suggestion
            cat "supabase/docker/.env.example" "../../.env.example" > $env_file
            echo "ENABLE_EMAIL_AUTOCONFIRM=true" >> $env_file
            echo "API_SUPABASE_URL=http://kong:8000" >> $env_file
```